packages/expo-av/android/build.gradle

```diff
@@ -72,6 +103,31 @@ dependencies {
 
   implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:${safeExtGet('kotlinVersion', '1.6.10')}"
 
+  //noinspection GradleDynamicVersion
+  implementation 'com.facebook.react:react-native:+'
+
+  compileOnly 'com.facebook.fbjni:fbjni:0.2.2'
+  extractHeaders 'com.facebook.fbjni:fbjni:0.2.2:headers'
```

Comment:
please check if `prefab` can save these.

Reply:
Unfortunately, I couldn't get that working - I had several issues when trying to get rid of all that _"extract JNI / headers"_ stuff from `build.gradle` 😞

Reply:
alright, let's keep it. please let me know if you've finished working on this pr and ready for review 😄

Reply:
I think it's ready (not planning any more changes), I updated the PR description 😉